### PR TITLE
fixed links in keep track of notebooks

### PR DESCRIPTION
**Pull request by jakubczakon**

### CODE REVIEW

**Review by kamil-kaczmarek — DISMISSED** [review]
Looks good!

File: docs/keep-track-of-jupyter-notebooks/index.rst [hunk 1]
```diff
@@ -272,3 +268,19 @@ OR
...
 
     <iframe width="720" height="420" src="https://www.youtube.com/embed/8qmz2yIndOw" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+.. |Notebooks in the project| raw:: html
+
+    <a href="https://ui.neptune.ai/shared/onboarding/notebooks" target="_blank">Notebooks in the project</a>
+
+.. |share a Notebook| raw:: html
+
+    <a href="https://ui.neptune.ai/shared/onboarding/n/neural-style-tutorial-c96dce51-409a-4b1b-8dbf-c47d52868d9b/9a7f6736-8794-44f0-9060-cf1b451d92d9" target="_blank">share a Notebook</a>
```
Comment (kamil-kaczmarek) [inline docs/keep-track-of-jupyter-notebooks/index.rst:278]: This link leads to 404 error

**Review by kamil-kaczmarek — CHANGES_REQUESTED** [review]
One link does not work (check comment).
Besides this looks good!

File: docs/keep-track-of-jupyter-notebooks/index.rst (hunk 1 above)
Comment (jakubczakon) [reply docs/keep-track-of-jupyter-notebooks/index.rst]: fixed/updated

Comment (kamil-kaczmarek) [reply docs/keep-track-of-jupyter-notebooks/index.rst]: I see that it works now.

**Review by kamil-kaczmarek — APPROVED** [review]

**Pull request merged**